x-pack/legacy/plugins/task_manager/index.js

```diff
@@ -16,7 +16,7 @@ export function taskManager(kibana) {
         enabled: Joi.boolean().default(true),
         max_attempts: Joi.number()
           .description('The maximum number of times a task will be attempted before being abandoned as failed')
-          .min(0) // no retries
+          .min(1)
```

Comment:
Just a qq for my understanding: I assume this is 1 now because the attempts logic counts the initial attempt as an attempt

Reply:
That's correct, the `attempts` attribute on a scheduled task will now increase when claiming a task instead of at the end of an execution. This makes the `attempts` consider timeouts now as well.